app/main/views/templates.py

```diff
@@ -922,37 +918,55 @@ def letter_template_attach_pages(service_id, template_id):
 
     form = PDFUploadForm()
     error = {}
+    letter_attachment_image_url = None
+    attachment_page_count = None
     if form.validate_on_submit():
+        upload_id = uuid.uuid4()
         try:
-            return _process_letter_attachment_form(service_id, template, form)
+            return _process_letter_attachment_form(service_id, template, form, upload_id)
         except LetterAttachmentFormError as e:
             error = e.as_error_dict()
+            attachment_page_count = error.get("attachment_page_count", 0)
+            letter_attachment_image_url = (
+                url_for(
+                    "no_cookie.view_invalid_letter_attachment_as_preview",
+                    service_id=service_id,
+                    file_id=upload_id,
+                ),
+            )[0]
```

Comment:
This looks like a mistake? We're making a one element tuple and then getting them first element from it.

I think the hanging comma was a typo that turned this into a tuple, and then `[0]` was a workaround for it. But we should just be able to remove it

```suggestion
            attachment_page_count = error.get("attachment_page_count", 0)
            letter_attachment_image_url = url_for(
                "no_cookie.view_invalid_letter_attachment_as_preview",
                service_id=service_id,
                file_id=upload_id,
            )
```

Reply:
Ahh, good catch, I was wondering why it's being wrapped in a tuple and somehow missed that we were the ones explicitly doing it 🙈